### ISSUE TITLE
Render placeholders as bold text in render.py

### ISSUE THREAD

**Issue by vitorhcl** [tldr-pages/tldr#11994]
I think thid could be achieved by replacing "{{" and "}}" by the HTML tags "<b>" and "</b>"

**Comment by kbdharun** [comment]
> I think thid could be achieved by replacing "{{" and "}}" by the HTML tags `<b>` and `</b>`.

Thanks for the proposal, while this does sound good on paper, this would be problematic in pages where we need to do placeholder escaping to display double brackets `{{ }}` along with the rendered placeholder. So we need to carefully proceed with this issue to avoid edge cases like this.

**Comment by Managor** [comment]
Can you explain this a bit further. What is render.py and why do you want to render placeholders in bold?

**Comment by kbdharun** [comment]
> Can you explain this a bit further. What is render.py and why do you want to render placeholders in bold?

Render.py (under `/scripts` directory) is a Python script which is used to create the tldr-book PDFs of all possible languages (some languages are ignored due to ligature rendering issues in the generated pages).

Tldr-book PDFs can be downloaded from the latest GitHub release. It allows you to view all pages offline without any clients and also allows you to print a specific cheat sheet or fully print it to keep in your physical collection or local library.

Coming back to this issue, I think vitorhcl has suggested placeholder braces to be bold to apply some stylized effect to the placeholder similar to our CLI clients and also to distinguish them from the rest of the command example.
